src/app/not-found.tsx

```
@@ -1,8 +1,375 @@
 export default function NotFound() {
   return (
-    <div>
-      <h1>Not Found</h1>
+    <div className="w-full flex flex-col justify-center items-center h-screen">
+      <svg
+        width="248"
+        height="180"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+        className="opacity-50 ml-6"
+      >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)